### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by td6904**
Merge of navbar v1 to branch 'dev' -  font under buttons on top right not yet set. Global SCSS containing fonts added in a different branch waiting for merge.

### CODE REVIEW

File: src/app/app-routing.module.ts
Comment (Benja-san) [inline src/app/app-routing.module.ts:1]: d'you still need the demoPageComponent?
I don't see any route for your homepage component

File: src/app/app.component.html [hunk 1]
```diff
@@ -1 +1,3 @@
+<app-navbar></app-navbar>
+<app-homepage></app-homepage>
 <router-outlet></router-outlet>
```
Comment (Benja-san) [inline src/app/app.component.html:3]: wrap the outlet into a <main> tag ?

File: src/app/navbar/navbar.component.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<nav>
+  <div class="chefs-compass">
```
Comment (Benja-san) [inline src/app/navbar/navbar.component.html:2]: This logo is a link to home page or not ?

File: src/app/navbar/navbar.component.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    <img class="nav-logo" src="/assets/main-logo.svg" alt="nav-logo" />
+    <p class="slogan">Global flavours from your kitchen</p>
+  </div>
+  <div class="search-module">
+    <input type="text" placeholder="Dishes..." />
+    <button type="submit"><img src="/assets/mag-glass.png" /></button>
+    <div class="mini-burger">
```
Comment (Benja-san) [inline src/app/navbar/navbar.component.html:9]: If this code is a placeholder, remove it for now cause it does not display anything

File: src/app/navbar/navbar.component.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      <div class="line"></div>
+      <div class="line"></div>
+      <div class="line"></div>
+    </div>
+  </div>
+  <div class="button-menu">
```
Comment (Benja-san) [inline src/app/navbar/navbar.component.html:15]: This should be a nav tag wrapping an ul tag with li tags for each link

File: src/app/navbar/navbar.component.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    <div>
+      <button>
```
Comment (Benja-san) [inline src/app/navbar/navbar.component.html:17]: I think those button are links to another pages so remove button tags into link tags

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/app/navbar/navbar.component.html [hunk 6]
```diff
@@ -1,41 +1,43 @@
...
-  </div>
-  <div class="button-menu">
-    <div>
-      <button>
+  </div> -->
+
+  <nav class="button-menu">
+    <ul>
+      <li>
+        <link />
```
[the remark this answers is not in the document]
Comment (td6904) [reply src/app/navbar/navbar.component.html:22]: Which tag are you referring to?

Nav is used for the navigation menu of each link, then each list item is in a list followed by a link tag to wrap the image and p. Personally I thought they were better as buttons. Am I better just to leave it as img and p tags and not wrap them ?

**Review by Benja-san — APPROVED** [review]

**Review by remy-rm — APPROVED** [review]

**Pull request merged**